orquesta/tests/unit/conducting/mistral/test_workflow_split.py

```diff
@@ -39,9 +41,11 @@ def test_split(self):
             ("task7", 2),
         ]
 
-        self.assert_spec_inspection(wf_name)
```

Comment:
Where is the spec inspection done now?  I don't see this performed in the WorkflowConductorMock's assert_conducting_sequences?

Reply:
done